test/DynamoCoreTests/DynamoCoreTests.csproj

```diff
@@ -142,4 +142,12 @@
     <Content Include="Crypto\AnAlteredFile.txt" />
     <Content Include="Crypto\AnImportantFile.txt" />
   </ItemGroup>
+  <Target Name="CopyDisableADPTestDependencies" AfterTargets="Build">
+      <Copy SourceFiles="$(PkgMicrosoft_Diagnostics_Runtime)\lib\net461\Microsoft.Diagnostics.Runtime.dll" DestinationFolder="$(OutputPath)/../../../test/packages" />
```

Comment:
so I don't see this mentioned in the PR description, what is this test/packages folder?

Reply:
I added comments in the PR description